manifests/base.pp

```diff
@@ -1,5 +1,8 @@
 # common things for sysctl
 class sysctl::base {
+
+  warning('This class will be deprecated in a future release, you should use ::sysctl instead.')
+
```

Comment:
Thus should probly be followed by an include. So we don't duplicate code.

Reply:
Why not instantiate the sysctl class inside sysctl::base class.

class sysctl::base {
   warning( )
   class {'::sysctl': }